weblogic-batch/scripts/check-for-jms-message.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+cd /apps/oracle/scripts
+
+# load variables created from setCron script if needed
+source /apps/oracle/env.variables
+
+if [ -z "$1" ]; then
+    echo `date`": Unable to confirm if process is already running on - no queue name passed as a parameter."
+    exit 1
+fi
+QUEUE_NAME=$1
+
+JNDI_QUEUE_NAME=${JMS_SERVER_NAME}@${QUEUE_NAME}
```

Comment:
Instead of setting these new local vars, you could just reference the existing vars when running java below.